pkg/operator/ceph/operator.go

```diff
@@ -101,6 +107,22 @@ func (o *Operator) cleanup(stopCh chan struct{}) {
 	o.clusterController.StopWatch()
 }
 
+func (o *Operator) updateOperatorLogs() error {
```

Comment:
Why returning an error if you always silence the errors?

Reply:
added return